main/glossary/README.md

```diff
@@ -219,7 +221,7 @@ See [`E()`](#e) above.
 Part of an [offer](#offer) specifying how the offer can be cancelled/exited. There are three values:
 - `onDemand: null`: (Default) The offering party can cancel on demand.
 - `waived: null`: The offering party can't cancel and relies entirely on the smart contract to promptly finish their offer.
-- `afterDeadline`: The offer is automatically cancelled after a deadline, as determined by its timer and deadline properties.
+- `afterDeadline: {…}`: The offer is automatically cancelled after a deadline, as determined by its `deadline` and `timer` properties.
```

Comment:
Maybe expand the nested record to show the timer and deadline?

Reply:
I opted instead for a link to https://agoric.com/documentation/zoe/api/zoe.html#proposals-and-payments .